lib/streamlit/elements/map.py

```diff
@@ -123,9 +123,9 @@ def map(
 
         Parameters
         ----------
-        data : pandas.DataFrame, pandas.Styler, pyarrow.Table, numpy.ndarray,\
-            pyspark.sql.DataFrame, snowflake.snowpark.dataframe.DataFrame,\
-            snowflake.snowpark.table.Table, Iterable, dict, or None
+        data : pandas.DataFrame, pandas.Styler, pyarrow.Table, pyspark.sql.DataFrame,\
+             snowflake.snowpark.dataframe.DataFrame, snowflake.snowpark.table.Table,\
+             Iterable, dict, or None
```

Comment:
```suggestion
            snowflake.snowpark.dataframe.DataFrame, snowflake.snowpark.table.Table,\
            Iterable, dict, or None

```

Reply:
Tiny nit: The extra, leading whitespace isn't technically necessary and it might look a bit more natural to keep the indenting consistent. The extra blank line is optional if it makes it easier to read.